tests/e2e/tidbcluster/serial.go

```diff
@@ -1008,4 +1009,121 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			klog.Info("success to check auto scale-in tidb to 2 replicas")
 		})
 	})
+
+	ginkgo.Context("[Verify: Upgrading Operator from 1.0.6", func() {
+		var oa tests.OperatorActions
+		var ocfg *tests.OperatorConfig
+		var version string
+
+		ginkgo.BeforeEach(func() {
+			version = os.Getenv("RELEASED_VERSION")
```

Comment:
Is this necessary as we will set it in the next line?

Reply:
updated.